src/tests/sqlsmith/tests/frontend/mod.rs

```diff
@@ -249,7 +249,7 @@ pub fn run() {
     let args = Arguments::from_args();
     let env = Arc::new(setup_sqlsmith_with_seed(0).unwrap());
 
-    let num_tests = 512;
+    let num_tests = 256;
```

Comment:
Avoid timeout for now. Properly fix later: https://github.com/risingwavelabs/risingwave/issues/6941